packages/sync-actions/src/products.js

```diff
@@ -121,9 +122,10 @@ function createProductMapActions(
 }
 
 function moveMasterVariantsIntoVariants(
-  before: Object,
-  now: Object
+  objectBefore: Object,
+  objectNow: Object
```

Comment:
I think we should keep the existing names because we use them everywhere else. That's why I suggest to revert these changes.

Reply:
i just changed it so that who reads the code could follow up easily 
plus i kept same names with this line
`[before, now] = copyEmptyArrayProps(objectBefore, objectNow)`

if i reverted it, it will be like below, the same object changed
`function moveMasterVariantsIntoVariants(
  before: Object,
  now: Object
): Array<Object> {
  [before, now] = copyEmptyArrayProps(before, now)`

what do you think?!

Reply:
@MarkusAzer Where can I see this change? How about pushing your changes, then it's easier to review them? 😄

Reply:
@katmatt sorry forgot to push, now its up